test/e2e/network/service_latency.go

```diff
@@ -89,13 +88,6 @@ var _ = SIGDescribe("Service endpoints latency", func() {
 		dSorted := durations(d)
 		sort.Sort(dSorted)
 		n := len(dSorted)
-		if n < minSampleSize {
```

Comment:
I actually would prefer to keep this. It seems to me that gating error rate is different than checking sample size. Even when the error rate is too high, this test would continue to calculate latency percentiles. So I would like to see a separate check to make sure the sample size is good enough.

This also helps ensure the sample quality in case `totalTrials` is changed to a lower value in the future.

Reply:
@MrHohn Thanks for your review, that makes sense.
Let's keep this check :-)